pkg/helm_actions/rollback_release.go

```diff
@@ -0,0 +1,23 @@
+package helm_actions
```

Comment:
`_` aren't appreciated in package names :) https://blog.golang.org/package-names
Maybe, name the package `helmcatalog/actions`. Later on, everything we add further with respect to Helm in the catalog associated can go into `helmcatalog/`

Reply:
Agree. I would suggest something like:

```
/pkg/helmcatalog/install/install.go
/pkg/helmcatalog/release/rollback.go
/pkg/helmcatalog/release/list.go
/pkg/helmcatalog/release/upgrade.go
/pkg/helmcatalog/template/template.go
/pkg/helmagent/agent.go # should this be a separate package?
etc
```
Packages are great.  And I assume some tests eventually!

Reply:
updated package path to `helm`